setup.sh

```diff
@@ -19,6 +19,19 @@ declare -r CONFIGS_PATH="configs"
 
 . src/kwio.sh --source-only
 
+function echo_n_run()
+{
+  echo "$@"
+  eval "$@ >/dev/null"
```

Comment:
Right. It is very similar, but `echo_n_run` redirects the output of the command to `/dev/null` and `cmd_manager` prints the command.

I also kind of feel like we should avoid using `src/*` code in `setup.sh` (but I know we already use kwlib's `complain` in someplaces)...

Reply:
As you said, we are using `kwio.sh` . Honestly, I don't see any huge problem in using `kw` libraries in the `setup.sh`, actually, I like the idea because our libraries are tested and we can avoid code duplication. What is your main argument for not using kw libraries?

About the lack of support for `/dev/null/` in the `cmd_command` function, I think you can easily add a "NO_CMD_OUTPUT" support in the `cmd_command`. We can take advange of this new option inside `kw` in the way that we can implement a `--verbose` and `non-verbose` options.